dsls/sql/src/main/java/org/apache/beam/dsls/sql/transform/BeamBuiltinAggregations.java

```diff
@@ -383,6 +384,144 @@ public T result(KV<BigDecimal, Long> accumulator) {
     }
   }
 
+  static class VarAgg implements Serializable {
+    long count; // number of elements
+    double sum; // sum of elements
+
+    public VarAgg(long count, double sum) {
+      this.count = count;
+      this.sum = sum;
+    }
+  }
+
+  public static final class Var<T> extends BeamSqlUdaf<T, KV<BigDecimal, VarAgg>, T> {
```

Comment:
based on here, https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/sql/fun/SqlAvgAggFunction.java#L44

Input type and Output type should be same.
However, Hive's AVG, VAR_POP can get IntType and return DoubleType. (Avg(IntType) -> DoubleType). Do we need to think of this situation or just follow calcite?